input/pagecontent/gaps-in-care-reporting.md

```diff
@@ -104,7 +104,7 @@ This section describes the profiles used for Gaps in Care Reporting and how they
 
 The [care-gaps](OperationDefinition-care-gaps.html) operation returns a [Parameters](https://www.hl7.org/fhir/parameters.html) resource that contains zero or more `parameter` with document bundle for each patient for which a Gaps in Care report is calculated. The bundle SHALL conform to the [DEQM Gaps In Care Bundle Profile]. A Gaps In Care Bundle SHALL contain a Composition entry, which uses the [DEQM Gaps In Care Composition Profile].
 
-The [DEQM Gaps in Care Composition Profile] builds on the base FHIR Composition resource, where its type code is constrained to a fixed LOINC code to identify the Composition as a Gaps in Care Report. The `subject` of a Gaps In Care Composition is required, it is used to reference the patient, [QI Core Patient], the Gaps in Care Report is for. The Gaps In Care Composition SHALL contain one to many section(s). Each `section` has a `focus` element that references an Individual MeasureReport for a specific measure. All Individual MeasureReport referenced SHALL be for the same patient specified in the Composition `subject`. Each `section` SHALL also contain one or more `entry` of DetectedIssue using the [DEQM Gaps In Care DetectedIssue Profile] for the measure regardless of its gap status (e.g., open or closed).
+The [DEQM Gaps in Care Composition Profile] builds on the base FHIR Composition resource, where its type code is constrained to a fixed LOINC code to identify the Composition as a Gaps in Care Report. The `subject` of a Gaps In Care Composition is required, it is used to reference the patient, [QI-Core Patient], the Gaps in Care Report is for. The Gaps In Care Composition SHALL contain one to many section(s). Each `section` has a `focus` element that references an Individual MeasureReport for a specific measure. All Individual MeasureReport referenced SHALL be for the same patient specified in the Composition `subject`. Each `section` SHALL also contain one or more `entry` of DetectedIssue using the [DEQM Gaps In Care DetectedIssue Profile] for the measure regardless of its gap status (e.g., open or closed).
```

Comment:
QICore Patient here is a specific QI-Core profile. Change to "QICore Patient"  (no space) instead of "QI-Core" to match the actual profile name.